src/test/java/teammates/test/pageobjects/AdminAccountManagementPage.java

```diff
@@ -92,4 +92,9 @@ public boolean isTableVisible() {
         }
         return result;
     }
+
+    public void waitForPageToFinishLoadingOnNonDevEnvironment() {
+        By currentPageEntryCountSpan = By.id("currentPageEntryCount");
+        waitForElementPresence(currentPageEntryCountSpan, 180);
```

Comment:
3 minutes might be too large. If I recall correctly, google app engine requests must return in 1 minute otherwise error 500 is returned.